stdlib/LinearAlgebra/src/qr.jl

```diff
@@ -813,12 +817,12 @@ ldiv!(A::QRPivoted{T}, B::StridedVector{T}) where {T<:BlasFloat} =
     vec(ldiv!(A,reshape(B,length(B),1)))
 ldiv!(A::QRPivoted{T}, B::StridedVecOrMat{T}) where {T<:BlasFloat} =
     ldiv!(A, B, min(size(A)...)*eps(real(float(one(eltype(B))))))[1]
-function ldiv!(A::QR{T}, B::StridedMatrix{T}) where T
+function _wide_qr_ldiv!(A::QR{T}, B::StridedMatrix{T}) where T
```

Comment:
I'm not sure, but 4 lines above this one, does the `vec` allocate? Maybe replace by
```julia
function ldiv!(A::QRPivoted{T}, B::StridedVector{T}) where {T<:BlasFloat}
     ldiv!(A, reshape(B, length(B), 1))
    return B
end
```
a pattern, that we use frequently here?

Reply:
`vec` doesn't allocate:
```julia
julia> a = [1 2; 3 4]
2×2 Array{Int64,2}:
 1  2
 3  4

julia> vec(a)[1] = 5
5

julia> a
2×2 Array{Int64,2}:
 5  2
 3  4
```